### PR TITLE
finished test

### PR DESCRIPTION
**Pull request by stacyburris**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/repeatedWord/repeated-word.test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const repeatedWord = require('../repeatedWord');
```
Comment (adrianhuebner) [inline javascript/code-challenges/repeatedWord/repeated-word.test.js:3]: This line is causing an error to occur, it needs to be:
```
const repeatedWord = require('./repeatedWord');
```
